src/Closet.java

```diff
@@ -0,0 +1,17 @@
+
+/**
+ * @author Joselyn Vasquez
+ */
+public interface Closet extends VirtualClosetManagerKernel {
```

Comment:
If I am following your logic correctly, this should be called `VirtualClosetManager` You can rename the proof of concept to whatever you like. 

```suggestion
public interface VirtualClosetManager extends VirtualClosetManagerKernel {
```